CS_alphabet/CS_alphabet/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CS_alphabet
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите текст:");
```

Comment:
Лучше указать язык ввода
```suggestion
            Console.WriteLine("Введите текст на русском языке:");
```